index.html

```diff
@@ -4,8 +4,15 @@
     <meta charset="UTF-8" />
     <title>ProviderKeeper</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="node_modules/@waves/signer/dist/signer.min.js"></script>
+    <script src="dist/provider-keeper.umd.js"></script>
   </head>
   <body style="height: 100%">
+    <script type="text/javascript">
```

Comment:
type can be omited, it's the default value. I didn't see this for a long time)))